pori_python/ipr/main.py

```diff
@@ -218,8 +225,12 @@ def clean_unsupported_content(upload_content: Dict, ipr_spec: Dict = {}) -> Dict
             ].get("kbCategory", "")
 
     for row in upload_content["kbMatches"]:
-        del row["kbContextId"]
-        del row["kbRelevanceId"]
+        for stmt in row["kbMatchedStatements"]:
+            del stmt["kbContextId"]
+            del stmt["kbRelevanceId"]
+            # del stmt["variantType"]
+            # del stmt["kbVariant"]
+            # del stmt["kbVariantId"]
```

Comment:
ah, no I need to remove these lines. These lines would remove the fields from the statement - not the variant the statement belongs to - but are not needed since other changes prevented them from being added in the first place.

Reply:
gotcha